packages/vapor/scss/components/header.scss

```diff
@@ -3,7 +3,7 @@
     width: 100%;
     height: $header-height;
     color: var(--white);
-    background-color: var(--navy-blue-80);
+    background: linear-gradient(270deg, var(--grape-purple-70) 0, var(--navy-blue-80) 65%);
```

Comment:
We already have a gradient variable in the [`colors.scss`](https://github.com/coveo/react-vapor/blob/master/packages/vapor/scss/colors.scss#L207) file. Its not the same one, but we could add a new variable in there for this one